docs/pieces/apps/clickup.mdx

```diff
@@ -16,5 +16,45 @@ title: 'Clickup'
       <Card title="Create Task">
         Create a new task in a ClickUp workspace and list
       </Card>
+      <Card title="Create Folderless List">
+        Create a new folderless list in a ClickUp workspace and spaces
+      </Card>
+      <Card title="Get List">
+        Get a list in a ClickUp
+      </Card>
+      <Card title="Get Space">
+        Get a space in a ClickUp workspace
+      </Card>
+      <Card title="Get Spaces">
+        Get all spaces in a ClickUp workspace
+      </Card>
+      <Card title="Get Task Comments">
+        Get all comments on a ClickUp task
+      </Card>
     </CardGroup>
     
+
+## How to get a list ID
```

Comment:
Sorry for not mentioning that in the docs, but these docs are auto generated from piece folder.

You can create clickup.mdx in same directory and add the custom part below.

Example: https://github.com/activepieces/activepieces/blob/main/packages/pieces/apps/src/lib/discord/discord.mdx